pydicom/charset.py

```diff
@@ -78,7 +78,7 @@
 default_encoding = "iso8859"
 
 
-def decode_string(value, encodings):
+def decode_string(value, encodings, delimiters):
```

Comment:
Could you expand the docstring with  Parameters and Returns sections?

Reply:
Yes, will do this - although I consider it an internal function.

Reply:
Done.

Reply:
I like documenting functions (in case you couldn't tell :wink:), since it helps me get it right in my head what a function is supposed to be doing and gives others a good starting point to understand the code. 

For example, I had to go through and try and find what the typical types/values of all three parameters should be before I could look at reviewing the code.

Reply:
I'm completely with you on that - what I actually wanted to say, is that I don't want a user to consider this an API function, so that we are free to change it at will. I also don't want it to prepend with an underscore, as we use it from outside the module - so I guess I would like some "internal" marker here, that prevents it from showing up in the generated docs.